.github/workflows/snyk_push.yml

```
@@ -0,0 +1,27 @@
+---
+name: Snyk Clojure Merge
```

(nit) to match the PR workflow name
```suggestion
name: Clojure Synk Push Check
```